ldms/python/ldmsd/ldmsd_yaml_parser

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+import os, sys
+import yaml
+import errno
+import argparse
+from ldmsd.parser_util import *
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(
+        description="LDMS Monitoring Cluster Configuration")
+    parser.add_argument("--ldms_config", metavar="FILE", required=True,
+                        help="The ldmsd load balance domain configuration file. "
+                        "This will not start the maestro "
+                        "load balancer")
+    parser.add_argument("--generate-config-path", metavar="STRING", required=False,
+                        default=False)
```

Comment:
Maybe you want to add the help string for `--generate-config-path`?

Reply:
Updated in [9b4113a](https://github.com/ovis-hpc/ldms/pull/1461/commits/9b4113a2e362def38eca1543eb71d2e326483e58)